src/connector/src/sink/doris_starrocks_connector.rs

```diff
@@ -143,6 +143,7 @@ impl HeaderBuilder {
         self
     }
 
+    /// Only use in Starrocks
```

Comment:
How does user to enable partial_update on starrocks sink? I think they can reuse the same with options on user side, but just set different header for different implementations.

Reply:
like create sink with(partial_update = 'true');